README.md

```diff
@@ -178,6 +178,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Tech Kunja Pvt. Ltd.](http://techkunja.com.np/) | Patan, Lalitpur | Web design, web development, AI, graphics design and consulting services
 - [Tech One Global Pvt. Ltd.](https://www.techoneglobal.com/) | Naxal, Kathmandu | Technology solutions to maximize your ROI and secure your organization
 - [Tech101](https://www.tech101.com.np/) | Kamaladi, Kathmandu | Web design, web development, graphics design and digital marketing
+- [Techybug](https://www.techybug.com/) | Bharatpur, Chitwan | WWeb Designing, Graphics Designing, Application Development, Hosting and Domain Service and many more
```

Comment:
Hello @sijannregmi thank you for the PR. 

The website doesn't align with the description. I see the website is a tech blog? Also, there is no contact information available in the website.

Sorry, this is an invalid entry.